cvat/apps/dataset_manager/tests/test_formats.py

```diff
@@ -484,6 +576,498 @@ def test_frames_outside_are_not_generated(self):
+            'YOLO 1.1': 'yolo',
+            'ImageNet 1.0': 'imagenet_txt',
+            'CamVid 1.0': 'camvid',
+            'WiderFace 1.0': 'wider_face',
+            'VGGFace2 1.0': 'vgg_face2',
+        }
+
+        # get formats
+        dump_formats = dm.views.get_export_formats()
+
+        for dump_format in dump_formats:
+            if dump_format.ENABLED:
+                dump_format_name = dump_format.DISPLAY_NAME
+
+                if dump_format_name != "Datumaro 1.0":
+                    continue
+                # TODO skip failed formats
+                if dump_format_name in [
+                    "CVAT for video 1.1",
+                    "YOLO 1.1",
+                    "ImageNet 1.0",
+                    "Datumaro 1.0",
+                ]:
+                    continue
+
+                print("dump_format_name:", dump_format_name)
+
+                # create task
+                images = self._generate_task_images(3)
+                task = self._create_task(tasks["main"], images)
+
+                 # create annotations
+                if dump_format_name in ["MOT 1.1", "MOTS PNG 1.0", "PASCAL VOC 1.1", "Segmentation mask 1.1", "TFRecord 1.0", "YOLO 1.1", "ImageNet 1.0", "WiderFace 1.0", "VGGFace2 1.0"]:
+                    self._create_annotations(task, dump_format_name, "dafault")
+                else:
+                    self._create_annotations(task, dump_format_name, "random")
+
+                task_id = task["id"]
+                task_ann = TaskAnnotation(task_id)
+                task_ann.init_from_db()
+
+                data_datumaro = {
+                    "dataset": None,
+                    "annotations": None,
+                }
+
+                for type_file in ("dataset", "annotations"):
+                    # dump file
+                    url = f"/api/v1/tasks/{task_id}/{type_file}"
+                    data = {
+                        "format": dump_format_name,
+                        "action": "download",
+                    }
+                    file_zip_name = osp.join(osp.dirname(__file__), 'assets', f'{test_name}_{type_file}_{dump_format_name}.zip')
+                    self._download_file(url, data, self.admin, file_zip_name)
+                    self._check_downloaded_file(file_zip_name)
+
+                    # extract zip
+                    folder_name = osp.join(osp.dirname(__file__), 'assets', f'{test_name}_{type_file}_{dump_format_name}')
+                    with zipfile.ZipFile(file_zip_name, 'r') as zip_ref:
+                        zip_ref.extractall(folder_name)
+                    data_datumaro[type_file] = datumaro.components.project.Dataset.import_from(folder_name, importer_format_name[dump_format_name])
+
+                # equals dataset vs annotations
+                compare_datasets(self, data_datumaro["dataset"], data_datumaro["annotations"])
+
+    @skip("Fail")
+    def test_api_v1_tasks_annotations_dump_and_upload_with_datumaro_many_jobs(self):
```

Comment:
Please clarify, what is the problem here? A task with multiple jobs is a regular case.

Reply:
Test of check CVAT for images is passed, but CVAT for video is failed due to #2923 and #2945 bugs.